src/AppChooser/AppButton.vala

```diff
@@ -15,22 +15,23 @@ public class AppChooser.AppButton : Gtk.ListBoxRow {
 
         var icon = new Gtk.Image () {
             gicon = app_info.get_icon () ?? new ThemedIcon ("application-default-icon"),
-            icon_size = Gtk.IconSize.DND
+            icon_size = Gtk.IconSize.LARGE
         };
 
         var name = new Gtk.Label (app_info.get_display_name ()) {
             ellipsize = Pango.EllipsizeMode.END
         };
 
-        var grid = new Gtk.Grid () {
-            column_spacing = 6,
-            margin = 3,
-            margin_start = 6,
-            margin_end = 6
+        var box = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 6) {
+            margin_top = 3,
+            margin_end = 6,
```

Comment:
I think we can replace manually setting these margins and the icon size above with the `rich-list` class from Granite